cpp/src/fil/treelite_import.cu

```diff
@@ -525,8 +531,8 @@ void tl2fil_common(forest_params_t* params,
 
   params->num_cols = model.num_feature;
 
-  ASSERT(param.sigmoid_alpha == 1.0f, "sigmoid_alpha not supported");
-  params->global_bias = param.global_bias;
+  ASSERT(model.sigmoid_alpha == 1.0f, "sigmoid_alpha not supported");
+  params->global_bias = model.base_scores[0];
```

Comment:
Do we want to add a runtime check here to confirm that all `model.base_scores` values are equal? Thinking about the possibility that folks would get unexpected results if they provide a vector bias and we only use the 0th element without further checks.